core/domain/classifier_services.py

```diff
@@ -356,3 +356,151 @@ def delete_classifier_training_job(job_id):
         classifier_models.ClassifierTrainingJobModel.get(job_id))
     if classifier_training_job_model is not None:
         classifier_training_job_model.delete()
+
+def get_classifier_exploration_mapping_from_model(mapping_model):
+    """Gets a classifier exploration mapping domain object from a classifier
+    exploration mapping model.
+
+    Args:
+        mapping_model: ClassifierExplorationMappingModel. Classifier Exploration
+            mapping instance in datastore.
+
+    Returns:
+        classifier_exploration_mapping: ClassifierExplorationMapping. Domain
+            object for the classifier exploration mapping.
+    """
+    return classifier_domain.ClassifierExplorationMapping(
+        mapping_model.id,
+        mapping_model.classifier_id)
+
+def get_classifier_exploration_mapping_by_id(mapping_id):
+    """Gets a classifier exploration mapping by a mapping_id.
+
+    Args:
+        mapping_id: str. ID of the classifier exploration mapping.
+
+    Returns:
+        classifier_exploration_mapping: ClassifierExplorationMapping. Domain
+            object for the classifier exploration mapping.
+
+    Raises:
+        Exception: Entity for class ClassifierExplorationMapping with id not
+            found.
+    """
+    classifier_exploration_mapping_model = (
+        classifier_models.ClassifierExplorationMappingModel.get(mapping_id))
+    classifier_exploration_mapping = (
+        get_classifier_exploration_mapping_from_model(
+            classifier_exploration_mapping_model))
+    return classifier_exploration_mapping
+
+def get_classifier_from_exploration_attributes(exp_id, exp_version,
+                                               state_name):
+    """Gets the classifier object from the exploration attributes.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. The exploration version.
+        state_name: str. The name of the state to which the classifier
+            belongs.
+
+    Returns:
+        classifier: ClassifierData. Domain object for the Classifier model.
+
+    Raises:
+        Exception: Entity for class ClassifierExplorationMapping with id not
+            found.
+        Exception: Entity for class ClassifierData with id not found.
+    """
+    mapping_id = exp_id + str(exp_version) + state_name
```

Comment:
@prasanna08 . Since we have a delete_classifier_exploration_mapping(), doesn't it make sense to have a get_classifier_exploration_mapping_by_id() too to? I didn't really use that function anywhere, I was using it within tests.